manyfews/calculations/zentra.py

```diff
@@ -166,26 +180,8 @@ def zentraReader(backTime, stationSN, token):
         VPD,
         RH,
     ]
-    zentraDataKeys = [
-        "local date",
-        "time stamp",
-        "solar radiation",
-        "precipitation",
-        "wind direction",
-        "wind speed",
-        "guest speed",
-        "vapor pressure",
-        "atmospheric pressure",
-        "air temperature",
-        "max precipitation Rate",
-        "RH senor temperature",
-        "VPD",
-        "Relative Humidity",
-    ]
-
-    zentraDataSet = dict(zip(zentraDataKeys, zentraDataSum))
 
-    return zentraDataSet
+    return zentraDataSum
```

Comment:
Sorry, this isn't what I meant: I think it makes it less clear.

We already have an object type (ZentraReading) that can store all this information in a way that is easy to access using property names. So can we just create a ZentraReading object at this point instead of passing around lists of numbers?

Reply:
accepted and changed.